CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+
+### Added
+
+- Added App.capture_print, App.end_capture_print, Widget.capture_print, Widget.end_capture_print https://github.com/Textualize/textual/issues/2952
```

Comment:
I feel we should add the same methods to `Screen` too.

Reply:
The screen is a Widget, so it will inherit those methods.

Reply:
Damn, good point. I keep forgetting that. Amusingly screen and widget both now have a notify method. Note to self: remove the wrapper on Screen.